odoo/cli/i18n.py

```diff
@@ -128,7 +128,8 @@ def run(self, cmdargs):
     def _get_languages(self, env, language_codes, active_test=True):
         # We want to log invalid parameters
         Lang = env['res.lang'].with_context(active_test=False)
-        languages = Lang.search([('iso_code', 'in', language_codes)])
+        languages = Lang.search(['|', ('iso_code', 'in', language_codes),
+                                 ('code', 'in', language_codes)])
```

Comment:
import and use `Domain`

Reply:
Change the commit title to:

> [IMP] core: support res.lang.code in i18n cli

There are some spelling/grammar mistakes in the message. Your commit body can wrap at 72 chars, wrapping it as 50 chars makes it unnecessarily narrow.

Suggestion:

> The i18n CLI only supports importing/exporting PO files using the ISO
> lang (e.g. fr_FR), but the PO files sometimes use the shorter lang
> code (e.g. fr instead of fr_FR) as filename.
> 
> This work adds support for the shorter lang code in the i18n CLI, for
> consistency.